backend/database/factories/UserFactory.php

```diff
@@ -15,11 +15,19 @@ class UserFactory extends Factory
     public function definition()
     {
         return [
-            'name' => $this->faker->name(),
+            'id' => $this->faker->unique()->uuid(),
+            'user_name' => $this->faker->userName(),
+            // password attribute is mutated when being created, see backend\app\Models\User.php
+            // 'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password <-- this is default from app creation
```

Comment:
remove comments since this is unnecessary

```suggestion
            // 'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password <-- this is default from app creation
```